src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -421,14 +421,10 @@ private void Initialize(AnalyzerExecutor analyzerExecutor, DiagnosticQueue diagn
                 if (_lazyInitializeTask == null)
                 {
                     // Set initializeTask to be a cancelled task.
-                    var tcs = new TaskCompletionSource<int>();
-                    tcs.SetCanceled();
-                    _lazyInitializeTask = tcs.Task;
+                    _lazyInitializeTask = Task.FromCanceled(new CancellationToken(true));
```

Comment:
Consider using named parameters here and below.

Reply:
Done

Reply:
Done